packages/fxa-settings/src/pages/ResetPassword/CompleteResetPassword/container.tsx

```diff
@@ -72,7 +77,19 @@ const CompleteResetPasswordContainer = ({
     navigate('/reset_password_with_recovery_key_verified');
   };
 
-  const handleNavigationWithoutRecoveryKey = () => {
+  const handleNavigationWithoutRecoveryKey = async (
+    accountResetData: AccountResetData
+  ) => {
+    if (accountResetData.verified && isWebIntegration(integration)) {
```

Comment:
Is there plans to do this for other integrations like Sync?

Reply:
hmm re-reading the jira again, I think this patch should include sync.  I'll make that change.  Thanks.